src/poetry_to_uv/poetry.py

```diff
@@ -54,7 +54,12 @@ def _process_poetry_export_command(self, poetry_response: bytes) -> PoetryResolv
                 continue
 
             if "@" in dependency_version:
-                dependency, version = dependency_version.split("@")
+                try:
+                    dependency, version = dependency_version.split("@", maxsplit=1)
+                    dependency = dependency.strip()
+                    version = version.strip()
+                except ValueError as e:
+                    print(f'{e} with {dependency_version}')
```

Comment:
Yeah, it's probably best to handle this in another way, but I guess it could be fine to just alert the user that we failed to convert this specific dependency to the UV format.